CHANGELOG.md

```diff
@@ -38,6 +38,23 @@ IMPROVEMENTS:
   * `golang.org/x/sync` v0.5.0 -> v0.6.0
   * `google.golang.org/api` v0.154.0 -> v0.163.0
 
+## v0.19.1
+
+FEATURES:
+  * Added `normalize_bound_audiences` as a flag that can be enabled to ignore the trailing slash at the end of each bound audience. Bound audiences that differ by only a trailing slash are treated as equivalent. This feature is set to `false` by default.
```

Comment:
I would call this an Improvement not a Feature (see my comment on the main changelog PR)